src/main/java/appeng/core/Registration.java

```diff
@@ -141,68 +138,24 @@ public void setRecipeInformation( File f, CustomRecipeConfig f2 )
 		this.customRecipeConfig = f2;
 	}
 
-	Registration()
-	{
-		this.recipeHandler = new RecipeHandler();
-	}
-
-	public Biome getStorageBiome()
-	{
-		return this.storageBiome;
-	}
-
-	public static void addBlockToRegister( Block b )
-	{
-		if( blocksToRegister == null )
-		{
-			throw new IllegalStateException( "Past the registration phase already!" );
-		}
-		blocksToRegister.add( b );
-	}
-
-	public static void addRecipeToRegister( IRecipe r )
-	{
-		if( recipesToRegister == null )
-		{
-			throw new IllegalStateException( "Past the registration phase already!" );
-		}
-		recipesToRegister.add( r );
-	}
-
-	public static void addItemToRegister( Item i )
-	{
-		if( itemsToRegister == null )
-		{
-			throw new IllegalStateException( "Past the registration phase already!" );
-		}
-		itemsToRegister.add( i );
-	}
-
-	public DimensionType getStorageDimensionType()
-	{
-		return this.storageDimensionType;
-	}
-
 	void preInitialize( final FMLPreInitializationEvent event )
 	{
-		// this.registerSpatial( false );
-
 		Capabilities.register();
 
 		final Api api = Api.INSTANCE;
 		final IRecipeHandlerRegistry recipeRegistry = api.registries().recipes();
 		this.registerCraftHandlers( recipeRegistry );
 
-		RecipeSorter.register( "AE2-Facade", FacadeRecipe.class, Category.SHAPED, "" );
-		RecipeSorter.register( "AE2-Shaped", ShapedRecipe.class, Category.SHAPED, "" );
-		RecipeSorter.register( "AE2-Shapeless", ShapelessRecipe.class, Category.SHAPELESS, "" );
+		// RecipeSorter.register( "AE2-Facade", FacadeRecipe.class, Category.SHAPED, "" );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)